frontend/simple/js/state.js

```diff
@@ -179,4 +201,11 @@ const actions = {
 const debouncedSave = debounce(dispatch => dispatch('saveSettings'), 500)
 
 store = new Vuex.Store({state, mutations, getters, actions})
+async function loadLastUser () {
+  let user = await db.loadCurrentUser()
+  if (user) {
+    store.dispatch('login', user)
+  }
+}
+loadLastUser()
```

Comment:
I moved this to `main.js` after realizing this needs to happen before routing is accessed . Its wrapped in a function for the purpose of async/await